src/cloud/project/project-conf-files_services.md

```diff
@@ -182,6 +184,141 @@ During the deployment process, {{site.data.var.ct}} checks installed service ver
 
 To maintain Magento store security, update installed software versions before they reach EOL.
 
+## Change service version
+
+You can upgrade the installed service version for compatibility with the Magento version deployed in your Cloud environment.
+
+You cannot downgrade the service version for an installed service directly. However, you can create a new service with the required version. See [Downgrade service version](#downgrade-version).
+
+Use the [Service versions](#service-versions) table to check service version compatibility by Magento version. Note that some service versions supported by {{ site.data.var.ee }} are not supported on {{ site.data.var.ece }}.
+
+{:.bs-callout-warning}
+You must submit a support ticket to change the service configuration on Pro Production and Staging environments.
+
+{:.bs-callout-info}
+If you change the Elasticsearch service version, check the Elasticsearch composer package for compatibility with the new version. See [Elasticsearch software compatibility]({{ site.baseurl}}/cloud/project/project-conf-files_services-elastic.html#elasticsearch-software-compatibility).
+
+### Upgrade installed service version
+
+You can upgrade the installed service version by updating the service configuration in the `services.yaml` file.
+
+1. Change the [`type`](#type) value for the service in the `.magento/services.yaml` file:
+
+   > Original service definition
+
+   ```yaml
+   mysql:
+       type: mysql:10.2
+       disk: 2048
+   ```
+
+   > Updated service definition
+
+   ```yaml
+   mysql:
+       type: mysql:10.3
+       disk: 2048
+   ```
+
+1. Add, commit, and push your code changes.
+
+   ```bash
+   git add -A
+   ```
+
+   ```bash
+   git commit -m "Upgrade MySQL from MariaDB 10.2 to 10.3."
+   ```
+
+   ```bash
+   git push origin <branch-name>
+   ```
+
+### Downgrade version
+
+You cannot downgrade an installed service directly. You have two options:
+
+-  Rename an existing service with the new version, which removes the existing service and adds a new one.
+
+-  Create a new service and save the data from the existing service.
+
+When you change the service version, you must update the service configuration in the `services.yaml` file, and update the relationships in the `.magento.app.yaml` file.
+
+{:.procedure}
+To downgrade a service version by renaming an existing service:
+
+1. Rename the existing service in the `.magento/services.yaml` file and change the version.
```

Comment:
```suggestion
1. Rename the existing service in the `.magento/services.yaml` file and change the version.

   {:.bs-callout-warning}
   Renaming an existing service replaces it and deletes all data. If you need to retain the data, create a new service instead of renaming the existing one.
```